src/lib.rs

```diff
@@ -76,6 +77,57 @@ struct AddressLineInfo {
     pub column: usize,
 }
 
+/// Types of symbols..
+#[derive(Clone, Copy)]
+pub enum SymbolType {
+    Unknown,
+    Function,
+    Variable,
+}
+
+/// The context of an address finding request.
+///
+/// This type passes additionl parameters to resolvers.
+#[doc(hidden)]
+pub struct FindAddrContext {
```

Comment:
we have CacheHolderOpts, so why not sticking to "Opts" vs "Context"?

Reply:
Agree